terraform/crates-io/impl/cloudfront-webapp.tf

```diff
@@ -125,3 +127,20 @@ resource "aws_route53_record" "webapp_apex" {
     evaluate_target_health = false
   }
 }
+
+resource "aws_cloudfront_response_headers_policy" "webapp" {
```

Comment:
Would it make sense to link to the [issue](https://github.com/rust-lang/crates.io/issues/5332) in a comment so that it's very obvious *why* we're doing this?

Reply:
Great idea! I've added a comment linking to the issue. :+1: